content/03.when-to-use-dl.md

```diff
@@ -1,51 +1,45 @@
 
-There are some types of problems in which using deep learning is strongly indicated over machine learning.
-Assuming a sufficient quantity of quality data is available, applications such as computer vision and natural language processing are likely to benefit from deep learning.
-In fact, these areas were the first to see significant breakthroughs through the application of deep learning [@doi:10.1145/3065386] during the recent deep learning revolution."
-For example, Ferreira et al. used deep learning to recognize individual birds from images [@doi:10.1111/2041-210X.13436].
-This problem was historically difficult but, by combining automatic data collection using RFID tags with data augmentation and transfer learning (explained in [Tip 5](#architecture)), the authors were able to use deep learning achieve 90% accuracy in several species.
-Other areas include generative models, in which new samples are able to be created based on the training data, and reinforcement learning, in which agents are trained to interact with their environments.
-In general, before using deep learning, investigate whether similar problems (including analogous ones in other domains) have been solved successfully using deep learning.
+Despite these limitations, deep learning is strongly indicated over traditional machine learning for certain research questions and problems.
+In general, these include problems that feature hidden patterns across the data, complex relationships, and interrelated variables.
+Problems in computer vision and natural language processing often exhibit exhibit these very features, which helps to explain why these areas were some of the first to experience significant breakthroughs during the recent deep learning revolution [@doi:10.1145/3065386]. 
+As long as data of sufficient quantity and quality is available (_i.e._ large amounts of accurate and labeled data), applications to areas of biology with related data characteristics, such as genetic medicine [@doi:10.3389/fpsyt.2018.00290], radiology [@doi:10.1007/s11604-018-0726-3], microscopy [@doi:10.1364/OPTICA.4.001437], and pharmacovigilance [@doi:10.1093/jamia/ocw180], are similarly likely to benefit from deep learning techniques.
+For example, Ferreira et al. used deep learning to recognize individual birds from images [@doi:10.1111/2041-210X.13436] despite this problem being very difficult historicallly. By combining automatic data collection using RFID tags with data augmentation and transfer learning (explained in [Tip 5](#architecture)), the authors were able to use deep learning to achieve 90% accuracy in several species.
+[This sentence feel like it comes out of nowhere - is there a way to 1) better integrate or 2) leave for a later tip?]Other research areas include generative models, in which new samples are able to be created based on the training data, and reinforcement learning, in which agents are trained to interact with their environments.
```

Comment:
Which sentence are you referring to here? The sentence on the bird recognition or the generative and RL models sentence?

Reply:
Thanks @Benjamin-Lee

Here I was referring to the sentence 
```suggestion
Other research areas include generative models, in which new samples are able to be created based on the training data, and reinforcement learning, in which agents are trained to interact with their environments.
```
To me, it feels as though it doesn’t connect clearly to the previous sentence/paragraph or directly to the material introduced earlier. Is there a way to add some “connective tissue” to add it in? Or find another place it potentially fits better?

Reply:
The reason I put that sentence there is that things like [style transfer](https://www.tensorflow.org/tutorials/generative/style_transfer) (making one picture have the style of a painting) are pretty much impossible to do with traditional ML right now*. If your goal is something like that, you need DL.

I think the issue lies in the juxtaposition of the example of well done DL based biology research (e.g. the bird identification example). What about

```suggestion
Other research areas where deep learning excels are generative modeling, in which new samples are able to be created based on the training data, and reinforcement learning, in which agents are trained to interact with their environments.
```

*that I know of